10-05/task_1/task1.rb

```diff
@@ -0,0 +1,25 @@
+require 'csv'
+
+nums_two_columns = ""
+nums_buffer1  = []
+nums_buffer2 = []
+
+File.open("int_nums1.txt").each do |line|
+  nums_buffer1 << line.chomp
+end
+
+File.open("int_nums2.txt").each do |line|
+  nums_buffer2 << line.chomp
+end
+
+nums_buffer1.each_index do |index1|
```

Comment:
there is also a method called `each_with_index`. You can use it, so you don't have to do 2 loops.
the actual code will look like this:
```ruby
nums_buffer1.each_with_index do |number, index|
  nums_two_columns << "|#{number.rjust(30)}|#{nums_buffer2[index].rjust(30)}|\n"
end
```